tests/pytests/unit/modules/test_vmctl.py

```diff
@@ -0,0 +1,265 @@
+    expected = {"changes": True, "console": "/dev/ttyp4"}
+    with patch.dict(vmctl.__salt__, {"cmd.run_all": mock_cmd}):
+        with patch("salt.modules.vmctl.status", mock_status):
+            res = vmctl.start("web1", bootpath="/bsd.rd", nics=2, disk="/disk.img")
+            mock_cmd.assert_called_once_with(
+                [
+                    "vmctl",
+                    "start",
+                    "web1",
+                    "-i 2",
+                    "-b",
+                    "/bsd.rd",
+                    "-d",
+                    "/disk.img",
+                ],
+                output_loglevel="trace",
+                python_shell=False,
+            )
+            assert expected == res
+
+
+def test_status():
+    """
+    Tests getting status for all VMs.
+    """
+    ret = {}
+    ret["stdout"] = (
+        "   ID   PID VCPUS  MAXMEM  CURMEM     TTY        OWNER NAME\n"
+        "    1   123     1    2.9G     150M  ttyp5       john web1 - stopping\n"
+        "    2   456     1    512M     301M  ttyp4       paul web2\n"
+        "    3     -     1    512M       -       -       george web3\n"
+    )
+    ret["retcode"] = 0
+    mock_cmd = MagicMock(return_value=ret)
+    expected = {
+        "web1": {
+            "curmem": "150M",
+            "id": "1",
+            "maxmem": "2.9G",
+            "owner": "john",
+            "pid": "123",
+            "state": "stopping",
+            "tty": "ttyp5",
+            "vcpus": "1",
+        },
+        "web2": {
+            "curmem": "301M",
+            "id": "2",
+            "maxmem": "512M",
+            "owner": "paul",
+            "pid": "456",
+            "state": "running",
+            "tty": "ttyp4",
+            "vcpus": "1",
+        },
+        "web3": {
+            "curmem": "-",
+            "id": "3",
+            "maxmem": "512M",
+            "owner": "george",
+            "pid": "-",
+            "state": "stopped",
+            "tty": "-",
+            "vcpus": "1",
+        },
+    }
+    with patch.dict(vmctl.__salt__, {"cmd.run_all": mock_cmd}):
+        assert expected == vmctl.status()
+
+
+def test_status_single():
+    """
+    Tests getting status for a single VM.
+    """
+    ret = {}
+    ret["stdout"] = (
+        "   ID   PID VCPUS  MAXMEM  CURMEM     TTY        OWNER NAME\n"
+        "    1   123     1    2.9G     150M  ttyp5       ringo web4\n"
+        "    2     -     1    512M       -       -       george web3\n"
+    )
+    ret["retcode"] = 0
+    mock_cmd = MagicMock(return_value=ret)
+    expected = {
+        "web4": {
+            "curmem": "150M",
+            "id": "1",
+            "maxmem": "2.9G",
+            "owner": "ringo",
+            "pid": "123",
+            "state": "running",
+            "tty": "ttyp5",
+            "vcpus": "1",
+        },
+    }
+    with patch.dict(vmctl.__salt__, {"cmd.run_all": mock_cmd}):
+        assert expected == vmctl.status("web4")
```

Comment:
What we expect should be on the right side of `==`.

Let's clean all the asserts to match this.

Reply:
Thanks Pedro, should be fixed now.